### PR TITLE
[wasm][debugger] await test server startup

### PR DESCRIPTION
**Pull request by lewing**
Await test test harness startup and start hoisting shared state above the tests

### CODE REVIEW

File: sdks/wasm/DebuggerTestSuite/Support.cs [hunk 1]
```diff
@@ -60,13 +60,13 @@ async Task OnMessage(string method, JObject args, CancellationToken token)
 				await eventListeners[method](args, token);
 		}
 
-		public async Task Ready (Func<InspectorClient, CancellationToken, Task> cb = null) {
+		public async Task Ready (Func<InspectorClient, CancellationToken, Task> cb = null, TimeSpan? span = null) {
```
Comment (kg) [inline sdks/wasm/DebuggerTestSuite/Support.cs]: int timeoutMs = 60 * 1000?

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Comment by lewing** [comment]
@monojenkins build failed

**Pull request merged**